### PR TITLE
Adding sales query by product type id

### PR DESCRIPTION
**Pull request by jcleyba**
@ManuelMGM

### CODE REVIEW

File: models/SaleDetail.js [hunk 1]
```diff
@@ -41,6 +47,62 @@ class SaleDetail extends Entity {
...
+          {
+            model: Product.dbModel,
+            where: { id_ProductType: productTypeId },
+          },
+        ],
+        order: [['createdAt', 'DESC']],
+        where,
+      });
+    } catch (e) {
+      console.log(e);
```
Comment (ManuelMGM) [inline models/SaleDetail.js]: cambiar .log pr .error

File: models/SaleDetail.js [hunk 2]
```diff
@@ -41,6 +47,62 @@ class SaleDetail extends Entity {
...
+
+      return e;
```
Comment (ManuelMGM) [inline models/SaleDetail.js]: throw (e)

**Review by ManuelMGM — APPROVED** [review]

File: models/SaleDetail.js (hunk 2 above)
Comment (jcleyba) [reply models/SaleDetail.js]: Listo

File: models/SaleDetail.js (hunk 1 above)
Comment (jcleyba) [reply models/SaleDetail.js]: Listo

**Pull request merged**